### ISSUE TITLE
For the issue that Chinese character cannot be read in database 

### ISSUE THREAD

**Issue by heyacherry** [britter/spring-boot-heroku-demo#6]
Hi , thanks for providing this nice demo to me , which make the spring boot app works in herku.

Howerver, when I type Chinese charater , it is fine to show on the page , but I can not read the record on database with cmd. It shows the err as below , do you have any good idea ?

hicherry::DATABASE=> select \* from record; ERROR: 
 character with byte sequence 0xe4 0xb8 0xad in encoding "UTF8" has no equivalent in encoding "WIN1252"

**Comment by britter** [comment]
@CaiYiLiang happy you like it! This is really only a minimal sample project not ready for production use. So I can only speculate. It looks like the chinese characters have been inserted at UTF-8 sequences into the table. But when you try to read them out, they can't be represented in the WIN1252 charset. What are you using to read from the data base? Maybe your data base client is not configured correctly.

**Comment by heyacherry** [comment]
Hi @britter 
Thanks for your detailed response , so kind of you.
I tried many ways to fix the ecode issue( WIN1252 ) , but it still not works. And I tried to remote-connect the postgreSQL database with pgAdmin3 , it can display Chinese characters well. As postgreSQL and deployment server are not my current focus point , so I just stop the step. 
Thanks for your demo and kind reply , as without the demo , I couldn't successfully deploy my apps on heroku (https://hicherry.herokuapp.com/).  Thanks again!
